actors/evm/src/lib.rs

```diff
@@ -252,10 +256,12 @@ fn handle_filecoin_method_input(method: u64, codec: u64, params: &[u8]) -> Vec<u
 }
 
 impl ActorCode for EvmContractActor {
+    type Methods = Method;
+    // TODO: Use actor_dispatch for this, maybe.
```

Comment:
Probably. We'll need to add `pre { }` and `post { }` block support. Well, really just pre.

```rust
actor_dispatch! {
    pre {
        if method > EVM_MAX_RESERVED_METHOD { .. }
    },
    Foo => Bar...
}
```

Could get tricky... but doable, I think.

Reply:
https://github.com/filecoin-project/builtin-actors/issues/966